src/Analyser/ResultCache/ResultCacheManager.php

```diff
@@ -789,10 +789,7 @@ private function getFileHash(string $path): string
 			return $this->fileHashes[$path];
 		}
 
-		$contents = FileReader::read($path);
-		$contents = str_replace("\r\n", "\n", $contents);
```

Comment:
here was some normalization going on. removing it did not break tests.

I assume the result cache is not meant to be shared across different computers, therefore it seems not necessary